agent/util-scripts/test-bin/rpmdev-vercmp

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+args="${@}"
+echo "${0} ${args}" >> ${_testlog}
+
+if [[ "${2}" == "not-greater-equal-1.0.0" ]]; then
+    _rc=12
+elif [[ "${2}" == "greater-equal-1.0.0" || "${2}" == "greater-1.0.0" ]]; then
+    _rc=11
+elif [[ "${2}" == "not-greater-1.0.0" ]]; then
```

Comment:
Do you really want to be checking `$2` for an exact match?  Wouldn't it be better to check just the (e.g.) "not-greater-equal" part?  (There's a reasonable shell syntax for that.)

Reply:
I feel more comfortable with the exact matches to avoid overlapping names causing a problem.